_maps/readme.md

```diff
@@ -4,29 +4,23 @@
 
 The list of items that are modular requiring adding/modifying to /tg/ maps:
 
-- Positronic Alert Consoles (inside Robotics area)
-- Command Vendor (inside Meeting area)
-- LustWish Vendor (inside Recreational/Dormitory area)
 - Barbershop Room (including the vendors, chairs, mirrors, and more)
-- NanoTrasen Consultant's Office
-- Blueshield Office
 - Cryopods (inside Recreational/Dormitory area)
-- Stasis Beds to Stasis Pods
-- Armory/Gun Change
-- Prison Change
-- Security Medical Room
-- Arrival Tram and Console (inside Arrival area)
+- Security:
+- - Add Ammo Workbench
+- - Add Gun Vendor
+- Armory:
+- - Replace Disablers With Smartgun
+- - Replace Shotguns With Shotgun Spawners
+- - Replace Energy Guns With MCRS
+- - Replace Laser Guns With CMGs
+- Security Medics Room:
+- - Security Medics Spawner
+- - Security Medics Locker
+- Arrival Docks:
+- - Arrival Shuttle Docking Port Resize
+- - Arrival Shuttle Console
 - Central Command Ferry Hangar (inside Arrival area)
-- Update roundstart_template variable on mining/public/labour stationary docks: mining/x > mining/skyrat, mining_common/x > mining_common/skyrat, labour/x > labour/skyrat , mining/large > mining/large/skyrat
```

Comment:
Won't this mean that those shuttles won't be using the Skyrat ones?

Reply:
No, this was never ever required, it's all code.

Reply:
Thats entirely false, it will spawn the TG template per shuttle ID.
Source: I had to troubleshoot and add this line because they, as I said, need to have their ID in the shuttle docks.

Reply:
This is not something we should be doing map wise. It should be code. So shame on thee.